### PR TITLE
Update TST + LC Docs

### PR DESCRIPTION
**Pull request by aswatt**
Adding steps to run Log Collector in TST to Log Collector docs.

### CODE REVIEW

**Review by KidiIT — APPROVED** [review]
All reviewed and successfully approved.🤓❤️🫶

**Comment by KidiIT** [comment]
All checked and approved🌚❤️

**Comment by KidiIT** [comment]
All done here! Thank you and you're welcome🙃❤️

File: docs/Troubleshooting-Tool.md
Comment (KidiIT) [inline docs/Troubleshooting-Tool.md:1]: All Reviewed and successfully approved by KIDI'S TECH your friend.

File: tools/LogCollector/OMS_Linux_Agent_Log_Collector.md
Comment (KidiIT) [inline tools/LogCollector/OMS_Linux_Agent_Log_Collector.md:1]: Reviewed all changes made and successfully approved the files. Thank you and you're welcome!

**Review by KidiIT — COMMENTED** [review]
I reviewed all the changes made and successfully approved the files. Thank you and you're welcome! from your best friend KIDI'S TECH.

**Review by KidiIT — APPROVED** [review]
Successfully approved✅

**Review by KidiIT — APPROVED** [review]
Reviewed and approved all the recent changes made on the two files.🌚❤️🫶

**Comment by KidiIT** [comment]
successfully reviewed new changes made on the file and approved all new changes made on the application to be completed today🤓❤️🫶

**Comment by KidiIT** [comment]
Thank you y'all and you're welcome😌❤️

**Review by KidiIT — APPROVED** [review]
All reviewed and approved🫶❤️❤️

**Review by KidiIT — COMMENTED** [review]
Successfully completed my final project on my own with the intention of creating an awesome project that will help people in their field to achieve their goals and dreams to make it easier for everyone to succeed without being discouraged or overwhelmed by the process of achieving their dreams in life and work hard for it help me to be successful in my future. self confidence is the best key to success in life and the most powerful part is that the process will continue and I am proud of myself for being able to accomplish this goals in my own time and career. Thank you for your support and contributions to my journey to achieve this goals🫶❤️✅

**Review by AaronRM — APPROVED** [review]

**Review by KidiIT — COMMENTED** [review]
All reviewed and approved🫶❤️

**Comment by KidiIT** [comment]
All checked and successfully completed

**Review by KidiIT — APPROVED** [review]
All reviewed and approved🫶❤️

**Review by KidiIT — COMMENTED** [review]
All reviewed, resolved and approved❤️🫶 Thank you and you're welcome🌚

**Review by KidiIT — APPROVED** [review]
All reviewed and approved🫶❤️ Thank you so very much for your time and consideration in our work with the team and for your patience in the process of making this happen for us and to make sure we have a good outcome in the future for the project and for the community❤️🫶 Thanks once again🙌❤️

**Pull request merged**